src/api/spec/features/beta/webui/login_spec.rb

```diff
@@ -7,17 +7,19 @@
 
   scenario 'login with home project shows a link to it' do
     login user
-    within('#personal-navigation') do
-      expect(page).to have_link('Home Project')
+    click_link('Places')
+    within('#navigation') do
+      expect(page).to have_link('Your Home Project')
```

Comment:
Why not do the same as in 5eede45cd8e3be7e434e3ec066916018c49100c0

Reply:
Good catch!

Reply:
Done.